myaws/ssm_parameter_env.go

```diff
@@ -51,9 +51,12 @@ func formatSSMParameterAsEnv(parameter *ssm.Parameter, prefix string, dockerForm
 	// The name of environment variable should be uppercase.
 	name := strings.ToUpper(flatten)
 	outputOptionName := ""
+	// Escape % character
+	value := strings.Replace(*parameter.Value, "%", "%%", -1)
+
 	if dockerFormat {
 		// Output in docker environment variables format such as -e KEY=VALUE
 		outputOptionName = "-e "
 	}
-	return fmt.Sprintf("%s%s=%s", outputOptionName, name, *parameter.Value)
+	return fmt.Sprintf("%s%s=%s", outputOptionName, name, value)
```

Comment:
Thank you for this.
I think `%` is not only the case of this bug.
We should use concatenate raw string values.

Reply:
Thank your for your review!!!

I fixed to concatenate key (with =) and raw value instead of escape '%'.
And I also fix `SSMParameterEnv`.  I use fmt.Print because joined key/value string doesn't need to be formatted.